### PR TITLE
RBAC Provider Operation Support

### PR DESCRIPTION
**Pull request by ArulvelMuralidharan**
RBAC Provider Operation Support

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -7,7 +7,7 @@ require (
 	github.com/Azure/go-autorest/autorest v0.9.0
 	github.com/Azure/go-autorest/autorest/date v0.2.0
 	github.com/google/uuid v1.3.0
-	github.com/microsoft/moc v0.11.0-alpha.2
+	github.com/microsoft/moc v0.10.29-alpha.3
```
Comment (madhanrm) [inline go.mod:10]: Are we going back on the versioning?

Comment (ArulvelMuralidharan) [reply go.mod:10]: Ragha said he was using this tag for his own testing. I think v0.10.29-alpha.3 is the latest tag

Comment (madhanrm) [reply go.mod:10]: Please create something like v0.11.0-alpha.3 and use that as latest.
v0.10 is old

Comment (ArulvelMuralidharan) [reply go.mod]: Resolved

File: services/security/security.go [hunk 2]
```diff
@@ -163,12 +163,64 @@ const (
 type Operation string
 
 const (
-	ReadAccess   Operation = "read"
-	WriteAccess  Operation = "write"
-	DeleteAccess Operation = "delete"
-	AllAccess    Operation = "all"
+	OBSOLETE_ReadAccess   Operation = "read"
```
Comment (raghavendra-nataraj) [inline services/security/security.go:166]: Dowe know if they are being used outside this repo?

Comment (ArulvelMuralidharan) [reply services/security/security.go:166]: Don't believe so. Only place I would think is moccli but I checked and they don't there.

**Review by raghavendra-nataraj — APPROVED** [review]

**Pull request merged**